include/ctr/coll.h

```diff
@@ -41,5 +41,69 @@ typedef struct TestVertex
     SVec3 interpolationPoint;
 } TestVertex;
 
+typedef struct DriverQuadblockCollData
+{
+    SVec3 driverPos;
+    s16 driverHitRadius;
+    s32 driverHitRadiusSquared;
+    SVec3 driverNextPos;
+    u16 collFlags;
+    u16 searchFlags;
+    s16 unk0;
+    u32 skipCollNoQuadFlagsMatch;
+} DriverQuadblockCollData;
+
+typedef union CollInputData
+{
+    DriverQuadblockCollData quadblock;
+} CollInputData;
+
+typedef struct CollDCache
```

Comment:
does this start at 0x1f800000 or 0x1f800108?

Reply:
0x1f800000